data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) 2017 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Drought Relief"
+	name `Drought relief to <planet>`
+	job
+	repeat
+	description `The farming world of <destination> is currently experiencing a drought. Deliver <cargo> to the planet by <date> for <payment>.`
+	cargo "drought relief" 25 2 .05
+	deadline
+	to offer
+		random < 10
+	source
+		government "Republic"
+	destination
+		distance 3 10
+		attributes "dirt belt"
+		attributes "farming"
+	on complete
+		payment 10000 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>. A group of <planet> citizens thank you and bid you farewell."
+
+
+mission "To Remembrance Day celebration [0]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Bring <fare> to <destination> by <date> for the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	to offer
+		random < 60
+		or
+			and
+				month == 12
+				day >= 21
+			month == 1
+			and
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment 1000 180
+		dialog "You wish your <passengers> the best of luck on <planet>. The children run off your ship in green face paint, and the parents pay you <payment>."
```

Comment:
>The trip is very likely completed before the actual Remembrance Day - should the children really be painted up, even if it's still not even the first of the coming year?

Go talk to the people who put Christmas decorations up the day after Halloween. 
That's my explanation for it. :)
It could also be that the celebration for Remembrance Day starts earlier than the day itself, but the biggest party is on the actual day.

Reply:
As long as the existing lore is matched, all is well.

>In these days when the Navy ensures peace throughout much of the galaxy, on many worlds the Alphas are treated as nothing more than a bogeyman story to frighten children. Because the most striking physical feature of the Alphas was their photosynthetic skin, on some worlds kids even run around wearing green face paint on Remembrance Day, trying to frighten people. But on other worlds, especially on the fringes of space where the Navy seldom patrols, the Alphas are still spoken of with fear.

Reply:
I'm not sure it makes sense for Remembrance Day to involve travel between planets. And, it especially doesn't make sense that Dirt Belt inhabitants would spend money on that sort of travel.

Reply:
I imagine that some people may travel for the purpose of seeing family (I'm sure there are *some* families that are spread across multiple planets), or perhaps just going to a planet that has a bigger celebration that they would like to partake in. Would it be acceptable if I kept the jobs but significantly reduced the offer rate as a representation of how only a small number of regional inhabitants are able to travel?